### PR TITLE
Refine tensorflow doc (a follow up of pr #1253 to meet code review)

### LINKED ISSUE #1211: add md docs for tensorflow support

### PR DESCRIPTION
**Pull request by yangw1234**
## What changes were proposed in this pull request?

Refine tensorflow doc. Meet code reivew in pr #1253 

## How was this patch tested?

do not need to test

## Related links or issues (optional)
fixed #1211

### CODE REVIEW

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-## **Loading a tensorflow model from tensorflow model file**
+## **Loading a Tensorflow model from Tensorflow model file**
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: Load TensorFlow model into BigDL

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-If you have a pre-trained tensorflow model saved in a ".pb" file. You load it
-into BigDL using `Model.load_tensorflow` api. 
+If you have a pre-trained Tensorflow model saved in a ".pb" file. You can load it
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: saved in ".pb" file, you can ...

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+into BigDL.
 
 For more information on how to generate
 the ".pb" file, you can refer to [A Tool Developer's Guide to TensorFlow Model Files](https://www.tensorflow.org/extend/tool_developers/).
 Specifically, you should generate a model definition file and a set of checkpoints, then use the [freeze_graph](https://github.com/tensorflow/tensorflow/blob/v1.0.0/tensorflow/python/tools/freeze_graph.py)
 script to freeze the graph definition and weights in checkpoints into a single file.
 
-**Generate model definition file and checkpoints in python**
+**Generate model definition file and checkpoints in Tensorflow**
 ```python
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: ***Scala***

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 4]
```diff
@@ -25,36 +25,37 @@ with tf.Session() as sess:
...
 
-**Freeze graph definition and checkpoints into a single ".pb" file in shell**
+**Freeze graph definition and checkpoints into a single ".pb" file**
 ```shell
 wget https://raw.githubusercontent.com/tensorflow/tensorflow/v1.0.0/tensorflow/python/tools/freeze_graph.py
 python freeze_graph.py --input_graph /tmp/model/test.pbtxt --input_checkpoint /tmp/model/test.chkp --output_node_names=output --output_graph "/tmp/model/test.pb"
 ```
 
-**Load tensorflow model in Scala**
+**Load Tensorflow model in BigDL**
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: ***Scala***

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 5]
```diff
@@ -25,36 +25,37 @@ with tf.Session() as sess:
...
 ```scala
 import com.intel.analytics.bigdl.utils._
+import com.intel.analytics.bigdl.nn.Module
 val path = "/tmp/model/test.pb"
 val inputs = Seq("Placeholder")
 val outputs = Seq("output")
-val model = TensorflowLoader.load(path, Seq("Placeholder"), Seq("output"), ByteOrder.LITTLE_ENDIAN)
+val model = Module.loadTF(path, Seq("Placeholder"), Seq("output"), ByteOrder.LITTLE_ENDIAN)
 ```
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: ***Python***

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 6]
```diff
@@ -25,36 +25,37 @@ with tf.Session() as sess:
...
+from bigdl.nn.layer import *
 path = "/tmp/model/test.pb"
 inputs = ["Placeholder"]
 outputs = ["output"]
 model = Model.load_tensorflow(path, inputs, outputs, byte_order = "little_endian", bigdl_type="float")
 ```
 ---
 
-## **Saving a BigDL Graph model to tensorflow model file**
+## **Saving a BigDL Graph model to Tensorflow model file**
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: Saving a BigDL functional model

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 7]
```diff
@@ -25,36 +25,37 @@ with tf.Session() as sess:
...
 
-You can also save a graph model to protobuf files so that it can be used in tensorflow inference.
+You can also save a graph model to protobuf files so that it can be used in Tensorflow inference.
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: save a BigDL functional model - and add a reference to functional API

File: docs/docs/ProgrammingGuide/tensorflow-support.md [hunk 8]
```diff
@@ -85,22 +86,23 @@ sigmoid = Sigmoid()(linear)
...
 
-# save it to tensorflow model file
+# save it to Tensorflow model file
 Model.save_tensorflow(model_original, [("input", [4, 10])], "/tmp/model.pb")
 ```
 
 
 ---
-## **Build model using tensorflow and train with BigDL**
+## **Build model using Tensorflow and train with BigDL**
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/tensorflow-support.md]: Build a TensorFlow model to run on BigDL

**Comment by yangw1234** [comment]
pr validation pass

**Pull request merged**